src/utils/helpers/structure/generateReviewSectionActions.ts

```diff
@@ -51,11 +51,11 @@ const actionDefinitions: ActionDefinition[] = [
   {
     // Show "start" when is new review submitted to consolidation (even for partial reviews)
     action: ReviewAction.canStartReview,
-    checkMethod: ({ reviewStatus, reviewLevel, isSecondReview, isPendingReview }) =>
-      reviewStatus === ReviewStatus.Pending &&
+    checkMethod: ({ reviewAssignmentStatus, reviewLevel, isSecondReview, reviewStatus }) =>
+      reviewAssignmentStatus === ReviewAssignmentStatus.Assigned &&
       reviewLevel > 1 &&
       !isSecondReview &&
-      isPendingReview,
```

Comment:
One thing I noticed is that `isPendingReview` is inconsistent between first review and consolidation. (It's present before starting for review, but not consolidation). This is the kind of thing that is causing us headaches!